packages/util-waiter/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "@aws-sdk/util-waiter",
+  "version": "1.0.0-rc.8",
+  "description": "Shared utilities for client waiters for the AWS SDK",
+  "dependencies": {
+    "tslib": "^1.8.0",
+    "@aws-sdk/abort-controller": "^1.0.0-rc.7"
```

Use pinned dependency. This would be updated on lerna publish.
```suggestion
    "@aws-sdk/abort-controller": "1.0.0-rc.7"
```